ansible/roles/kubeadm/tasks/redhat.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: remove versionlock for kubeadm packages
-  command: yum versionlock delete kubeadm
+  command: yum versionlock {{ '--disablerepo "*" --enablerepo offline' if offline_mode_enabled else '' }} delete kubeadm
```

Comment:
How are we not seeing these issues for RHEL and CentOS, does `yum versionlock` not try to download the metadata for those?